drafts/use-cases/4.obtaining-single-event.md

```diff
@@ -0,0 +1,59 @@
+## Obtaining a single event
+
+### Story
+As an application user
```

Comment:
As discussed, please keep both the abstract story plus the concrete example. Thanks